ipi/engine/forcefields.py

```diff
@@ -855,7 +855,7 @@ def __init__(
             )
         except ValueError:
             raise ValueError(
-                "ERROR: Reading sGDML model " + self.model + " file failed."
+                "ERROR: Reading sGDML model " + self.sGDML_model + " file failed."
```

Comment:
Is this at all related to the current PR?

Reply:
I think it is ok to leave it here but I agree this does not seem related to the current PR.